src/routes/controllers/articles.controller.js

```diff
@@ -147,6 +147,10 @@ export async function getArticle(req, res) {
           as: 'author',
           attributes: ['firstName', 'lastName', 'username'],
         },
+        {
```

Comment:
@Japhethca I included this because I realised get single article did not come with the comments of that belong to it, which would result in multiple queries to the API, I decided to add this at once in this PR